terraform/environment/modules/environment/outputs.tf

```
@@ -29,3 +29,7 @@ output "feedbackdb_security_group_id" {
 output "cluster_name" {
   value = aws_ecs_cluster.online-lpa.name
 }
+
+output "aws_db_instance_arn" {
+  value = var.account.always_on ? aws_db_instance.api[0].arn : ""
```

Do you think that something like N/A or not set , would be better than just the ""  ?

---

the blank has meaning (same as a null) - i'' try to set that now to null, good spot if it works :)
```suggestion
  value = var.account.always_on ? aws_db_instance.api[0].arn : null
```